src/classes/CW_AdminControllerTest.cls

```diff
@@ -0,0 +1,199 @@
+@isTest
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.